app/src/androidTest/java/com/veniosg/dir/test/acceptance/FileManagerActivityTest.java

```diff
@@ -38,6 +40,17 @@
     private final File testChildDirectory = new File(testDirectory, "testChildDir");
     private final File testCopyDestination = new File(testDirectory, "testCopyDestination");
     private final File testChildFile = new File(testDirectory, "testChildFile");
+    private final File testDirectoryWithZeroItems = new File(sdCardDir, "testDirWithZeroItems");
+    private final File testDirectoryWithOneItem = new File(sdCardDir, "testDirWithOneItem");
+    private final File testDirectoryWithMultipleItems = new File(sdCardDir, "testDirWithMultipleItems");
+
+    private final File zeroItemsContainerDir = new File(testDirectoryWithZeroItems, "zeroItemsContainerDir");
```

Comment:
What's the difference with the files above?

Reply:
Done!